Package

```diff
@@ -9,14 +9,26 @@ plugin_install_dirname=${plugin_name}
 #   2. Architecture specific packages we depend on
 #   3. Package manager specific packages we depend on
 requirements="fledge"
+os=$(cat /etc/os-release | grep -w ID | cut -f2 -d"=")
+
 
 case "$arch" in
 	x86_64)
 		;;
 	armv7l)
 		;;
 	aarch64)
-	    requirements="${requirements},python3-pandas"
+	         case "$os" in
+
+                ubuntu)
+                         echo "Package building for ${plugin_name} is not supported for Ubuntu aarch64!!"
```

Comment:
reason?

Reply:
Because there is no requirement to execute this plugin on aarch_64 Ubuntu 18.04 platform yet.